### PR TITLE
Go examples

### PR DESCRIPTION
**Pull request by ryannguyen16**
Adds Golang examples.

### CODE REVIEW

File: official/docs/go/address/create_adress.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package example
+
+import (
+    "os"
+
+    "github.com/EasyPost/easypost-go/v2"
+)
+
+func main() {
+	apiKey := os.Getenv("EASYPOST_API_KEY")
```
Comment (jchen293) [inline official/docs/go/address/create_adress.go:10]: remove extra tab, this also applies to other files

File: official/docs/go/address/create_and_verify_address.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            State:   "CA",
+            Zip:     "94104",
+            Country: "US",
+            Company: "EasyPost",
+            Phone:   "415-123-4567",
+        },
+        &easypost.CreateAddressOptions{},
+	)
+
+	fmt.Println(address)
```
Comment (jchen293) [inline official/docs/go/address/create_and_verify_address.go:27]: indentation is off here, please check other files as well

File: official/docs/go/customs_info/create_customs_info.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+					Description:   "T-shirt",
+					Quantity:      1,
+					Value:         10.00,
+					Weight:        5,
+					OriginCountry: "US",
+				},
+			},
+		},
+	)
+	fmt.Println(customsInfo)
```
Comment (jchen293) [inline official/docs/go/customs_info/create_customs_info.go:33]: nit: break a new line

**Review by jchen293 — APPROVED** [review]

File: official/docs/go/api_keys/retrieve_api_key.go [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package example
```
[the remark this answers is not in the document]
Comment (Justintime50) [reply official/docs/go/api_keys/retrieve_api_key.go]: Indeed, fixed

File: official/docs/go/customs_info/create_customs_info.go (hunk 3 above)
Comment (Justintime50) [reply official/docs/go/customs_info/create_customs_info.go]: Fixed

**Review by Justintime50 — APPROVED** [review]

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**